.ruby-version

```diff
@@ -1 +1 @@
-2.7.8
+3.2.2
```

Comment:
unf_ext-0.0.8.2-x64-mingw-ucrt requires ruby version < 3.2.dev, >= 3.1, which is
incompatible with the current version, ruby 3.2.2p53

Reply:
Looks like this was a Gemfile.lock update issue